src/main/scala/is/hail/rvd/RVD.scala

```diff
@@ -0,0 +1,120 @@
+package is.hail.rvd
+
+import is.hail.annotations.{MemoryBuffer, RegionValue, RegionValueBuilder}
+import is.hail.expr.Type
+import org.apache.spark.{Partition, SparkContext}
+import org.apache.spark.rdd.{AggregateWithContext, RDD}
+import org.apache.spark.storage.StorageLevel
+
+import scala.reflect.ClassTag
+
+
+object RVD {
+  def apply(rowType: Type, rdd: RDD[RegionValue]): ConcreteRVD = new ConcreteRVD(rowType, rdd)
+}
+
+case class PersistedRVRDD(
```

Comment:
I agree the names aren't great, but I don't think it is right to call them RVDs, either.  Neither this nor its parameters are RVDs: they are the RDD[RegionValue] constituents necessary to implement persisted RVDs.  Also, I don't want to make these RVDs because this is uses for both persist on RVD and OrderedRVD.